packages/CopyInput/src/CopyInput.js

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Input from "@paprika/input";
+import Popover from "@paprika/popover";
+import Button from "@paprika/button";
+import Toast from "@paprika/toast";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import extractChildrenProps from "@paprika/helpers/lib/extractChildrenProps";
+import CopyIcon from "@paprika/icon/lib/ArrowDown"; // TODO: Replace with actual icon when ready
```

Comment:
If you want, I can help you do this in this PR (though a follow-up PR will be required in the `wasabicons` repo). 

Otherwise it can be a separate sub-task / PR.

Reply:
Let's do it in a different subtask